test/e2e/specs-jetpack-calypso/wp-jetpack-connect-spec.js

```diff
@@ -108,6 +109,8 @@ describe( `Jetpack Connect: (${ screenSize })`, function() {
 			return await jetpackAuthorizePage.waitToDisappear();
 		} );
 
+		/* These flows are disabled via `jetpack/connect/site-questions` feature flag in Calypso */
+		/*
```

Comment:
I tested `step.skip()`  on these but [`mocha-steps`](https://www.npmjs.com/package/mocha-steps) doesn't support it. Only `describe.skip` is available from Mocha. :-/

Reply:
[There's `xstep` to disable.](https://github.com/rprieto/mocha-steps#example)

Reply:
Ah, good eye. Thanks!

Reply:
https://github.com/Automattic/wp-calypso/pull/34620/commits/72e1c61a4111d06c1abff2d67ba456b02bbaac31